### DISCUSSION TITLE
Prisma, Planetscale & CI/CD full flow

### DISCUSSION THREAD

**Discussion by ydennisy** [prisma/prisma#10301] [Q&A] [answered]
Hi!

I think it would be very helpful to outline the full flow using prisma, planetscale and CI/CD.

There are parts of information scattered across various docs and conversations, but from someone migrating away from firebase it would be excellent to see all this information in once place, specifically;
- how should destructive migrations work, if data needs to be moved based on a schema change?
- how best to use planetscale functionality for branching in a prisma flow?
- should environments be split as branches or databases?
- how best to integrate testing into this flow?
- execute all of the above in CI/CD such as GH actions?

Thanks in advance!
D

**Comment by austincrim** [comment, marked as answer]
Hi @ydennisy 👋, we are excited about the future of Planetscale + Prisma!

We appreciate your feedback and will definitely factor it in to future content and docs work. 

Would you mind taking a look at our [existing issues](https://github.com/prisma/docs/issues?q=is%3Aissue+is%3Aopen+planetscale) related to Planetscale docs? Feel free to chime in and let us know what specifically you would like to see covered.

**Reply by alexluong** [reply]
Hi there, I'm actually curious about how the migration flow works, specifically how PlanetScale's deploy request feature fits into the CI/CD pipeline. How can I make sure DB schema is updated as the new app version is deployed?

I'm coming from a NoSQL background so SQL migration process is a little new to me. Hope to learn more from you folks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
